src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -13274,6 +13274,35 @@ private static Vector512<T> CreateRemainderMaskVector512<T>(int count)
             throw new NotSupportedException();
         }
 
+        private static Vector128<float> ApplyScalar<TOperator>(Vector128<float> floats) where TOperator : IUnaryOperator<float, float> =>
+            Vector128.Create(
+                TOperator.Invoke(floats[0]), TOperator.Invoke(floats[1]), TOperator.Invoke(floats[2]), TOperator.Invoke(floats[3]));
+
+        private static Vector256<float> ApplyScalar<TOperator>(Vector256<float> floats) where TOperator : IUnaryOperator<float, float> =>
+            Vector256.Create(
+                TOperator.Invoke(floats[0]), TOperator.Invoke(floats[1]), TOperator.Invoke(floats[2]), TOperator.Invoke(floats[3]),
+                TOperator.Invoke(floats[4]), TOperator.Invoke(floats[5]), TOperator.Invoke(floats[6]), TOperator.Invoke(floats[7]));
```

Comment:
nit: Due to how the vector types are built and how these extract operations can have hidden cost, it'd be better (and a little simpler) to do this as:
```suggestion
            Vector256.Create(
                ApplyScalar(floats.GetLower()),
                ApplyScalar(floats.GetUpper())
            );
```

You can see with the below sharplab link, that it results in around 95 bytes less of codegen and much fewer insert/extracts: https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIAYACY8gOgCUBXAOwwEt8YLAJI8ovLrl5hcAbhrEAzE1IMAwgwDeNBjqZKAajDAZo5UgA4APADMANhGwYAfAwCCAB3e2AngGUw2LbYUAAUhsamFjb2ji4AboEcMACUDAC8LuEmUGbmLKqwjjAh2rpldg4Y+RC4IQm2SQDadAC6yWilZToVjtW19U3kbR3UXbo9Vao1dYkwjaTDnV0TfTMNcwptS8lyo7pLigxZ0KQArEjRlS4eXn4BQaHHUGcXE/GzqRlHRtkv+YUYYpLcoxSbTAZzVrtYHjUGrCGNIbQvZjFZTfqzeaLFHLOHotZNTbIsbdPHgzEobEkhho8nrRqnKkk2kY+lIJmosmspoAdi2OJ2SwOBh+J3Ol1ibk8Pn8gWCpDCoue4reDAhn0ySr+BRgRRKOLKNxl92CBMEAHEYBgADIQADuMFCyWJJKNdzloQhLEtGAAqp5HSFndtdgBfIA==

Of course we really just want to vectorize everything, but until that can happen this will help avoid some pessimizations.